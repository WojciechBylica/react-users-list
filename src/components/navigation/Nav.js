import { NavLink } from 'react-router-dom';
import './style.scss';

const Nav = () => (
  <nav>
    <NavLink className="navigation__link" to="/">
      Home
    </NavLink>
  </nav>
);

export default Nav;
