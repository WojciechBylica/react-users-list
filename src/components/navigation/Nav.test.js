import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '.';

it('should render "Home" link', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Nav />
    </BrowserRouter>
  );
  expect(tree).toMatchSnapshot();
});
