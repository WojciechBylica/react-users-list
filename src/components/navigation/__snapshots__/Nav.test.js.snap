// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render "Home" link 1`] = `
<nav>
  <a
    aria-current="page"
    className="navigation__link active"
    href="#/"
    onClick={[Function]}
  >
    Home
  </a>
</nav>
`;
